src/SFA.DAS.ApprenticeCommitments.Web/Pages/Apprenticeships/ConfirmYourEmployer.cshtml

```diff
@@ -27,23 +29,30 @@
             </div>
 
             <form id="employer-form-hide" method="post" action="">
-                <div class="govuk-form-group">
+                <div class="govuk-form-group @(ViewData.ModelState.IsValid ? "" : "govuk-form-group--error")">
                     <fieldset class="govuk-fieldset">
                         <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
                             <h1 class="govuk-fieldset__heading">
                                 Is this your employer?
                             </h1>
                         </legend>
 
+                        @if (ViewData.ModelState.ContainsKey(nameof(Model.EmployerConfirm)) && ViewData.ModelState[nameof(Model.EmployerConfirm)].Errors.Any())
```

Comment:
Is this doing anything different to the `asp-validation-for` helper?

e.g. https://github.com/SkillsFundingAgency/das-apprentice-commitments-web/blob/3385d706ed3da6c769d8b04d8861d0d59c555d1b/src/SFA.DAS.ApprenticeCommitments.Web/Pages/ConfirmYourIdentity.cshtml#L28

Reply:
Good spot, I missed that